### DISCUSSION TITLE
Foundry FoundMe error

### DISCUSSION THREAD

**Discussion by DCVglobalnetwork** [Cyfrin/foundry-full-course-cu#1716] [General]
Please Can anyone see what I am doing wrong ?

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/105791829/89c28346-5e78-415e-9663-dc5e89562f73)

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/105791829/10ab3b71-328a-4658-ba54-dd6f4f9c7e18)

**Comment by EngrPips** [comment]
Hello @DCVglobalnetwork, What did the compiler say When you hovered over the error?

Maybe the `FundMe` contract might expect an argument, but you didn't pass any in.
